src/Misc/layoutbin/darwin.svc.sh.template

```diff
@@ -7,7 +7,7 @@ user_id=`id -u`
 
 if [ $user_id -ne 0 ]; then
     echo "Must run as sudo"
-    exit 1
+    exit 5
 fi
```

Comment:
While testing on OSX found that ProcessInvoker always return exit code 0, even when the bash script returns 5. This same logic works fine on Linux. OSX ProcessExplorer also returns 0, if "bash" is invoked with a non-existent script file as a param (which should fail with exit code 127). It looks that there is some OSX specific ProcessExplorer issue - needs more investigation.